docs/topics/selectors.rst

```diff
@@ -1034,7 +1034,7 @@ described with CSS selectors.
 
 Parsel also simplifies adding your own XPath extensions.
 
-.. autofunction:: parsel.xpathfuncs.set_xpathfunc
+For more details about the XPath extensions, see the `Parsel Documentation <https://parsel.readthedocs.io/en/latest/usage.html#parsel.xpathfuncs.set_xpathfunc>`_.
```

Comment:
I think the link should just be attached to "adding your own XPath extensions". And I think you should be able to use a Sphinx reference for it instead of an explicitly written link.